sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -56,14 +56,14 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/fabiomestre/unified-runtime.git")
```

Comment:
This will be updated to point to the right commit / fork once the PR on the Unified Runtime repo is merged